client/me/purchases/remove-plan-dialog/get-plan-features.tsx

```diff
@@ -0,0 +1,257 @@
+
+export default function getPlanFeatures(
+	productSlug: string | undefined,
+	translate: ReturnType< typeof useTranslate >,
+	withinFirstYear: boolean,
+	hasDomain: boolean,
+	domainSlug: string
+): string[] {
+	if ( ! productSlug ) {
+		return [];
+	}
+
+	const isMonthlyPlan = isMonthly( productSlug );
+
+	/**
+	 * Define feature labels.
+	 */
+	const freeDomainFirstYear = String( translate( 'Your free domain for the first year' ) );
+	const freeDomain = String(
+		translate( 'Your free domain www.%(slug)s', {
+			args: { slug: domainSlug },
+		} )
+	);
+	const bestInClassHosting = String( translate( 'Best-in-class hosting' ) );
+	const addFreeSite = String( translate( 'An add-free site' ) );
+	const collectPayments = String( translate( 'The ability to collect payments' ) );
+	const emailCustomerSupport = String( translate( 'Unlimited customer support via email' ) );
+	const liveChat = String( translate( 'Access to live chat support' ) );
+	const earnAndRevenue = String( translate( 'The ability to earn and revenue' ) );
+	const premiumThemes = String( translate( 'Access to premium themes' ) );
+	const analyticsIntegration = String( translate( 'Google Analytics integration' ) );
+	const accessPlugins = String( translate( 'Access to more than 50,000 plugins' ) );
+	const seoTools = String( translate( 'Advanced SEO tools' ) );
+	const automatedBackups = String( translate( 'Automated site backupsand one-click restore' ) );
+	const sftpAndDatabase = String( translate( 'SFTP and Database access' ) );
+	const acceptPayments = String( translate( 'Accept payments in 60+ countries' ) );
+	const shippingCarriers = String( translate( 'Integration with top shipping carriers' ) );
+	const premiumDesign = String(
+		translate( 'Premium design options customized for online stores' )
+	);
+	const more = String( translate( 'and more' ) );
+
+	/**
+	 * Personal plan
+	 */
+	if ( isWpComPersonalPlan( productSlug ) ) {
+		// Yearly plan
+		if ( ! isMonthlyPlan ) {
+			// Without domain
+			if ( ! hasDomain ) {
+				return [
+					freeDomainFirstYear,
+					bestInClassHosting,
+					addFreeSite,
+					collectPayments,
+					emailCustomerSupport,
+				];
+			}
+
+			// With domain within first year
+			if ( withinFirstYear ) {
+				return [
+					freeDomain,
+					bestInClassHosting,
+					addFreeSite,
+					collectPayments,
+					emailCustomerSupport,
+				];
+			}
+
+			// With domain post first year
+			return [ bestInClassHosting, addFreeSite, collectPayments, emailCustomerSupport ];
+		}
+
+		// Monthly plan
+		return [ bestInClassHosting, addFreeSite, collectPayments, emailCustomerSupport ];
+	}
+
+	/**
+	 * Premium plan
+	 */
+	if ( isWpComPremiumPlan( productSlug ) ) {
+		// Yearly plan
+		if ( ! isMonthlyPlan ) {
+			// Without domain
+			if ( ! hasDomain ) {
+				return [
+					freeDomainFirstYear,
+					liveChat,
+					earnAndRevenue,
+					premiumThemes,
+					analyticsIntegration,
+					bestInClassHosting,
+					addFreeSite,
```

Comment:
Did you really mean to leave `collectPayments` off here, when it's included in all the ones below?